test/util.js

```diff
@@ -1,55 +1,86 @@
 'use strict';
 
-const fsMock = require('fs');
 const path = require('path');
+const del = require('del');
+const makeDir = require('make-dir');
+const parentModule = require('parent-module');
+const os = require('os');
+const fs = require.requireActual('fs');
 
-exports.absolutePath = str => path.join(__dirname, str);
+class TempDir {
+  dir: string;
 
-exports.mockStatIsDirectory = function mockStatIsDirectory(result) {
-  const stats = {
-    isDirectory: () => result,
-  };
+  constructor() {
+    const tempDir = fs.realpathSync(os.tmpdir());
+    const parent = parentModule();
```

Comment:
Hey @chrisblossom, could you please explain the significance of these 2 lines?

Reply:
I've [added some comments](https://github.com/davidtheclark/cosmiconfig/pull/127/files#diff-d8642304b1d7b6801be66d60ee3cbc5bR13). Please let me know if that clears things up.

Reply:
The idea is to have a consistent directory name for each test for things like ``jest --watch`` and other third party tools to play nicely.

Also, because ``tmp`` doesn't use ``realpathSync``, it will break any tests [using ``process.chdir``](https://github.com/davidtheclark/cosmiconfig/pull/127/files#diff-4b7ed0b193d73a1b1fa66c79c737b673R463).

Reply:
I see, thanks for explaining.